shell/browser/api/electron_api_cookies.cc

```diff
@@ -80,6 +80,9 @@ struct Converter<net::CookieChangeCause> {
                                    const net::CookieChangeCause& val) {
     switch (val) {
       case net::CookieChangeCause::INSERTED:
+        return gin::StringToV8(isolate, "inserted");
```

Comment:
Changing the value for `net::CookieChangeCause::INSERTED` would be a breaking change.

We would then release the commit as part of the next major version (not Electron 39) and you'd need to add the change to https://github.com/electron/electron/blob/main/docs/breaking-changes.md.

If we add any new values here, we should also update the docs: https://github.com/electron/electron/blob/4adfc212b31d09df83416d8a8b9bfef30e002862/docs/api/cookies.md?plain=1#L53-L60